src/simplewallet/simplewallet.cpp

```diff
@@ -1074,8 +1089,8 @@ bool simple_wallet::import_multisig(const std::vector<std::string> &args)
     fail_msg_writer() << tr("usage: import_multisig_info <filename1> [<filename2>...] - one for each other participant");
     return true;
   }
-  if (m_wallet->ask_password() && !get_and_verify_password())
-    return true;
+
+  SCOPED_WALLET_UNLOCK();
```

Comment:
I see `import_multisig_info` not working:
```
[wallet 5ACZH3]: import_multisig_info msinfo-1 msinfo-2
Wallet password: 
Password needed - use the refresh command
Multisig info imported                                          
```
which was fixed by inserting
```c++
    m_in_manual_refresh.store(true, std::memory_order_relaxed);
    epee::misc_utils::auto_scope_leave_caller scope_exit_handler = epee::misc_utils::create_scope_leave_handler([&](){m_in_manual_refresh.store(false, std::memory_order_relaxed);});
```
before the call to `m_wallet->import_multisig(info)`.